src/api/app/components/workflow_artifacts_per_step_component.rb

```diff
@@ -34,6 +37,33 @@ def call
 
   private
 
+  def artifacts_for_set_flag(parsed_artifacts)
+    capture do
+      list_of_flags(parsed_artifacts[:flags])
+    end
+  end
+
+  def list_of_flags(flags)
+    flags.each do |flag|
+      concat(flag_step_sentence(flag))
+    end
+  end
+
+  def flag_step_sentence(flag)
+    path_details = package_or_project_path(flag)
+
+    tag.li do
```

Comment:
Comparing with other artifacts, this is how it looks like:

![Screenshot 2023-10-26 at 10-14-47 Open Build Service](https://github.com/openSUSE/open-build-service/assets/2581944/1847821a-8e5a-421e-a628-288b7ad5f5c3)

I'd say there are too many **bold** words in the flag sentence.

It's also a bit hard for me to understand the sentence:
> Set flag tag: build, status: enable for [home:Admin/et](http://localhost:3000/package/show/home:Admin/et), repository: openSUSE_Tumbleweed, architecture: x86_64.

What about?

- Set flag **build** enabled on project/package  [home:Admin/et](http://localhost:3000/repositories/home:Admin/et) for repository openSUSE_Tumbleweed and architecture x86_64.

- Set flag **publish** disabled on project  [home:Admin](http://localhost:3000/repositories/home:Admin) for repository openSUSE_Tumbleweed and architecture x86_64.

I also propose that the links don't go to the project or package overview but to their repositories tab.

Reply:
This is done. @danidoni now the project/package link is pointing towards the repositories page.